base/grafana/grafana.ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    category: rbac
+    deploy: sourcegraph
+  name: grafana
+roleRef:
+  apiGroup: ""
+  kind: ClusterRole
+  name: grafana
+subjects:
+- kind: ServiceAccount
+  name: grafana
+  namespace: default
```

Comment:
We can't assume that all of our users are installing Sourcegraph into the `default` namespace. You need to make sure that `subjects.namespace` here is the namespace that `base/grafana/grafana.ServiceAccount.yaml` is using, otherwise, this won't work. I think we need to: 

1. document that this `subjects.namespace` field needs to be the same namespace as `base/grafana/grafana.ServiceAccount.yaml`
1. edit the installation instructions (configure.md, etc. ) to call this out beforehand
1. In https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/migrate.md, document that customers need to make sure to configure this field when upgrading to Sourcegraph 3.8

Reply:
done